src/main/java/science/icebreaker/account/RegistrationRequest.java

```diff
@@ -0,0 +1,29 @@
+package science.icebreaker.account;
+
+
+public class RegistrationRequest {
+
+    private Account account;
+
+    private Profile profile;
+
+
```

Comment:
Missing constructor

Reply:
Which one do you need? Isn't the default constructor enough?